adapters/InMobi/InMobiAdapter/GADMAdapterInMobiRewardedAd.m

```diff
@@ -98,8 +98,7 @@ - (void)loadRewardedAdForAdConfiguration:
 
 - (void)requestRewardedAd {
   // Converting a string to a long long value.
-  long long placement =
-      [_adConfig.credentials.settings[GADMAdapterInMobiPlacementID] longLongValue];
+  long long placement = [_placementIdentifier longLongValue];
```

Comment:
I don't think we need an extra `_placementIdentifier` instance var, we can just follow the previous approach.